templates/templates/base.html

```diff
@@ -94,6 +94,15 @@
         </div>
       </div>
     </div>
+    <div id="newsletter-signup" class="p-strip u-no-padding--top">
+      <div class="u-fixed-width">
+        <div class="p-notification--positive u-no-margin--bottom">
+          <div class="p-notification__content">
+            <p class="p-notification__message">Thank you for signing up for our newsletter!</p>
```

Comment:
Missing close functionality, I think just repeating this will work:
```suggestion
            <p class="p-notification__message">Thank you for signing up for our newsletter!<a href="#" onclick="location.href = document.referrer; return false;"><i class="p-notification__close">Close</i></a></p>
```